Presentation.Web/app/utility/kendoGridLauncher.ts

```diff
@@ -121,7 +129,7 @@ module Kitos.Utility.KendoGrid {
             return {
                 field: this.dataSourceName,
                 title: this.title,
-                attributes: [],
+                attributes: { "parentId": this.parentId },
```

Comment:
remember that `attributes` is an array... now it has changed to  an object... are you sure it should not be an array?... because if it is an array in other situations, the code might break.. perhaps you should then introduce a new custom property if that is the case and use that in stead